keps/sig-apps/2255-pod-cost/README.md

```diff
@@ -256,12 +275,11 @@ _This section must be completed when targeting beta graduation to a release._
 [supported limits]: https://git.k8s.io/community//sig-scalability/configs-and-limits/thresholds.md
 [existing SLIs/SLOs]: https://git.k8s.io/community/sig-scalability/slos/slos.md#kubernetes-slisslos
 
-## Implementation History
-
 
 ## Drawbacks
 
 
 ## Alternatives
 
-Similar behaviour can be achieved through the Operator Framework which however will take a lot more configuration and setup work and is not a built-in Kubernetes feature.
+-->
```

Comment:
An alternative we considered was adding an API field instead of the annotation.

Of course, leaving this to third-party implementations is also an option.

Reply:
not sure how leaving this to third-party implementations would work.